commands/game.py

```diff
@@ -0,0 +1,935 @@
+                server_id = server_id_map[server]
+                image_contents = []
+                title_info = titles_api[str(title_id)]
+                event_title = f"data/img/titles/{server}/{title_info['baseImageName'][server_id]}.png"
+                image_contents.append(event_title)
+                event = Image.open(image_contents[0])
+                event = event.resize((368, 80), Image.ANTIALIAS)
+                new_im.paste(event, (x_offset, 250), event)
+
+                tier = title_info['rank'][server_id]
+                if tier != 'none' and tier != 'normal' and tier != 'extra':
+                    tier_title = f'data/img/titles/{server}/event_point_{tier}.png'
+                    image_contents.append(tier_title)
+                    tier = Image.open(image_contents[1])
+                    tier = tier.resize((368, 80), Image.ANTIALIAS)
+                    new_im.paste(tier, (x_offset, 250), tier)
+                elif tier == 'normal' or tier == 'extra':
+                    tier_title = f'data/img/titles/{server}/try_clear_{tier}.png'
+                    image_contents.append(tier_title)
+                    tier = Image.open(image_contents[1])
+                    tier = tier.resize((368, 80), Image.ANTIALIAS)
+                    new_im.paste(tier, (x_offset, 250), tier)
+                x_offset += 525
+        else:
+            x_offset = 250
+            image_contents = []
+            title_info = titles_api[str(equipped_title_ids[0])]
+            server_id = server_id_map[server]
+            event_title = f"data/img/titles/{server}/{title_info['baseImageName'][server_id]}.png"
+            image_contents.append(event_title)
+            event = Image.open(image_contents[0])
+            event = event.resize((368, 80), Image.ANTIALIAS)
+            new_im.paste(event, (x_offset, 250), event)
+
+            tier = title_info['rank'][server_id]
+            if tier != 'none':
+                tier_title = f'data/img/titles/{server}/event_point_{tier}.png'
+                image_contents.append(tier_title)
+                tier = Image.open(image_contents[1])
+                tier = tier.resize((368, 80), Image.ANTIALIAS)
+                new_im.paste(tier, (x_offset, 250), tier)
+
+        file_name = f'{str(uuid.uuid4())}.png'
+        saved_file_path = f'data/img/imgTmp/{file_name}'
+        if not path.exists('data/img/imgTmp/'):
+            filepath = Path('data/img/imgTmp/')
+            filepath.mkdir(parents=True, exist_ok=True)
+        new_im.save(saved_file_path)
+        return file_name, saved_file_path
+
+    game_commands = SlashCommandGroup('game', 'game info related commands')
+
+    @game_commands.command(name='lookup',
+                           description='Searches a player on the specified server by ID')
+    async def game_lookup(self,
+                          ctx: discord.ApplicationContext,
+                          player_id: Option(int, "Player ID to lookup",
```

Comment:
Preference to avoid any parameters that need underscores

```suggestion
                          id: Option(int, "Player ID to lookup",
```